src/Microsoft.Health.MeasurementUtility/Microsoft.Health.MeasurementUtility.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Common utilities used by Microsoft Health.</Description>
-    <TargetFrameworks>$(SupportedFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(LegacyVersion);$(SupportedFrameworks)</TargetFrameworks>
```

Comment:
I take it there is some .NET Core or .NET Framework library that will use this?

Reply:
Thank you for reviewing! Yes, the Microsoft.Health.Events project, which targets .netstandard2.0, in medtech repo is going to leverage the TrackDuration() method from this project to be consumed by our ILogger<T> instances which emit metrics as logs.